cognite/client/_api/datapoints.py

```diff
@@ -322,6 +322,55 @@ def insert_multiple(self, datapoints: List[Dict[str, Union[str, int, List]]]) ->
         dps_poster = DatapointsPoster(self)
         dps_poster.insert(datapoints)
 
+    @overload
+    def delete_point(self, point: Dict[str, Union[int, str, datetime]]) -> None:
+        ...
+
+    @overload
+    def delete_point(self, point: Sequence[Dict[str, Union[int, str, datetime]]]) -> None:
+        ...
+
+    def delete_point(
+        self, point: Union[Dict[str, Union[int, str, datetime]], Sequence[Dict[str, Union[int, str, datetime]]]]
```

Comment:
To me it doesn't make much sense to allow `str` here; as these are all the "unprecise" or relative time offsets like "an hour ago" or "now".

Reply:
Agree. But this dict also contains the externslId, which has a string value.